source/includes/Concepts/concept.md

```diff
@@ -191,34 +186,32 @@ DELETE /concept/:target_concept_uuid?purge=true
 
 * ### List all concept mappings for a concept.
 
-    Retrieve all **concept mapping** sub resources of a **concept** resource by target_concept_uuid. Returns a 
-    `404 Not Found` status if concept mapping not exists. If user not logged in to perform this action, a `401 Unauthorized` status
-    returned.
+    Retrieve all **concept mapping** subresources of a **concept** resource by target_concept_uuid. Returns a 
+    `404 Not Found` status if concept mapping not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
 
 ```console
 GET /concept/:target_concept_uuid/mapping 
 ```
 
-* ### List concept mapping by it's UUID and parent concept UUID.
+* ### List concept mapping by its UUID and parent concept UUID.
     
-     Retrieve an **concept mapping** sub resources of a **concept** resource. Returns a 
-     `404 Not Found` status if concept mapping not exists. If you are not logged in to perform this action, a `401 Unauthorized` status
-     returned.
+     Retrieve a **concept mapping** subresources of a **concept** resource. Returns a 
+     `404 Not Found` status if concept mapping not exists. If you are not logged in to perform this action, a `401 Unauthorized` status returned.
      
 ```console
 GET /concept/:target_concept_uuid/mapping/:target_concept_mapping_uuid
 ```
 ## Create a concept mapping with properties
 
-* To Create a concept mapping sub resource for a specific concept resource you need to specify below attributes in the request body.
-If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* To Create a concept mapping subresource for a specific concept resource, you need to specify below attributes in the request body.
```

Comment:
```suggestion
* To create a concept mapping sub-resource for a specific concept resource, you need to specify below attributes in the request body.
```

Reply:
Okay @sanjulamadurapperuma I will make the changes 🙂